### ISSUE TITLE
Update manual 

### ISSUE THREAD

**Issue by poopgiggle** [SynerconTechnologies/FLA-User-Manual#4]
We need to get this done. Here's a candidate checklist of to-dos before we can release it.
- [x] Resize images to decrease the Brobdingnagian proportions of the PDF.
- [x] Take out Internet Sharing tutorial (maybe). (James)
- [x] Redo FLA Preview section to reflect new FLA Preview (Gavin)
- [x] Update section 6.2.1 note on automatic timezone adjustment, remove Gavin's PII (Gavin)
- [x] Update supported ECMs in section 8 (James)
- [x] Delete section 9 (James)
- [x] Redo section 10 to reflect new FLA Preview (James)
- [x] Resize images in section 11 (Gavin)
- [x] Consider removing 12.3 (duplicate of material from section 8) (Gavin)
- [x] Move "support mode" section to Troubleshooting (James)
- [x] Move "downloading drivers" to "Getting Started" section (Gavin)
- [x] Move benchtop download to appropriate part of section 8 (James)
- [x] Fix url control sequences (currently rendered literally as `\url{...}`) (James)
- [x] Fix email tags (Gavin)
- [x] Proofread (James)
- [x] Remove forensic writeblocking section. I think we've pretty much given up on that.
- [x] Fix chapter/section/other references. No more "chapter X" (James)
